src/main/java/com/salesforce/dynamodbv2/mt/context/MtAmazonDynamoDbContextProvider.java

```diff
@@ -18,11 +17,7 @@
 @FunctionalInterface
 public interface MtAmazonDynamoDbContextProvider {
 
-    Optional<String> getContextOpt();
-
-    default String getContext() {
-        return getContextOpt().orElseThrow(IllegalStateException::new);
-    }
+    String getContext();
```

Comment:
There is a pretty significant behavior change here, which is that it's now possible to put/get items in system context which was not possible before. Do we actually want that behavior?